src/containers/TimeseriesForRecordingView.tsx

```diff
@@ -37,9 +37,11 @@ const TimeseriesForRecordingView: FunctionComponent<Props> = ({ recordingId, rec
       <div>
         <TimeseriesViewNew
           recordingObject={recording.recordingObject}
+          recordingInfo={recording.recordingInfo}
           width={width}
           height={height}
           hither={hither}
+          opts={{channelSelectPanel: true}}
```

Comment:
This option controls whether the elec geom widget appears in a left panel, allowing user to select:

![image](https://user-images.githubusercontent.com/3679296/102085079-27881680-3de4-11eb-8f6b-4ad85795ea28.png)